client/signup/steps/domains/index.jsx

```diff
@@ -516,12 +518,26 @@ class DomainsStep extends React.Component {
 	};
 
 	getSubHeaderText() {
-		const { translate } = this.props;
+		const { flowName, siteType, translate } = this.props;
+		const onboardingSubHeaderCopy =
+			siteType &&
+			includes( [ 'onboarding-for-business', 'onboarding' ], flowName ) &&
```

Comment:
Do we need to restrict that to the onboarding flow? The default copy should take care of other flows.

Reply:
I wasn't 100% sure! Let's go for it! I'll ship this as is and follow up in a quick PR 👍